src/chain/shared_state.rs

```diff
@@ -222,6 +222,13 @@ impl SharedState {
         &self.our_members
     }
 
+    /// Returns an iterator over the members that have state == `Joined`.
+    pub fn our_present_members(&self) -> impl Iterator<Item = (&PublicId, &MemberInfo)> {
```

Comment:
Minor: either rename this method to `our_joined_members` or rename `MemberState::Joined` to `MemberState::Present` for consistency.